src/main/java/dev/jbang/cli/Init.java

```diff
@@ -116,13 +119,22 @@ public Integer doCall() throws IOException {
 		}
 
 		String renderedScriptOrFile = getRenderedScriptOrFile(tpl.fileRefs, refTargets, outDir, absolute);
-		info("File initialized. You can now run it with 'jbang " + renderedScriptOrFile
-				+ "' or edit it using 'jbang edit --open=[editor] "
-				+ renderedScriptOrFile + "' where [editor] is your editor or IDE, e.g. '"
-				+ Edit.knownEditors[new Random().nextInt(Edit.knownEditors.length)]
-				+ "'. If your IDE supports JBang, you can edit the directory instead: 'jbang edit . '"
-				+ renderedScriptOrFile + ". See https://jbang.dev/ide");
-
+		if (edit) {
+			info("File initialized. Opening editor for you. You can also now run it with 'jbang "
+					+ renderedScriptOrFile);
+			// TODO: quick hack that gets the job of opening editor done; but really should
```

Comment:
To me this actually isn't a hack but a completely valid way of doing things. An API _might_ be nice but could very well be overkill for something that (so far at least) isn't meant to be used as a library.

Reply:
its just thtat it prints out you can run with -b which is not the case for this ;)